src/ExceptionsServiceProvider.php

```diff
@@ -78,9 +82,9 @@ public function register()
             $assets = function ($path) use ($generator) {
                 return $generator->asset($path);
             };
-            $path = __DIR__.'/../resources/error.html';
+            $path = config('exceptions.views.html');
```

Comment:
I'd rather we didn't use any global functions here, in fact, with our dependencies, no such function exists.

Please resolve the configuration object from the container provided by the `$app` parameter. :)